imports/plugins/included/braintree/lib/collections/schemas/braintree.js

```diff
@@ -43,7 +43,8 @@ export const BraintreePayment = new SimpleSchema({
   },
   cardNumber: {
     type: String,
-    min: 16,
+    min: 12,
+    max: 19,
```

Comment:
The Maestro debit card was the most popular one I could find that used 19 digits, but I made this change based on [Error Code 81716](https://developers.braintreepayments.com/reference/general/validation-errors/all/node)